scripts/manage-imports.py

```diff
@@ -167,27 +173,31 @@ def main():
     load_config(configfile)
 
     from infogami import config
-    servername = config.get('servername', 'https://openlibrary.org')
 
     cmd = sys.argv[1]
-    args = sys.argv[2:]
+    args, flags = [], {'servername': config.get('servername', 'https://openlibrary.org')}
```

Comment:
Should this default servername be grabbed from or related to values in olsystem?